07/16/19
## Authors:
Truong, D.T., Tett, A., Pasolli, E., Huttenhower, C., and N. Segata
## Title:
Microbial strain-level population structure and genetic diversity from metagenomes
## Keywords:
Population genomics, microbiome, strain-level, metagenomics
## Geographic Coverage
Global, human gut
## Field Dates:
N/A
## Significance:
Development and application of StrainPhlAn, a tool for differentiating prokaryotic strains from metagenomes

## Notes:
StrainPhlAn allows for the identification of SNVs within a set of genes specific to a species allowing for strain identification.

In the human gut, rarely was a species in a sample dominated by more than one strain.

StrainPhlAn has an error rate less than 0.1% for per-nucleotide level strain variation on sequences with at least 2x coverage, lower than comparable tools.

StrainPhlAn is implemented within the MetaPhlAn2 python package.

Concluding hypotheses
* Strain-level identification can help resolve fine differences between populations
* Low strain heterogeneity is specifically discussed in reference to human guts, but may not always be true for other environments.
* Low strain heterogeneity may be a result of fine grained competition and may be associated with low incidences of HGT.
